test/TestUtils/TestProjectFixture.cs

```diff
@@ -0,0 +1,290 @@
+﻿using Microsoft.DotNet.Cli.Build;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace Microsoft.DotNet.CoreSetup.Test
+{
+    public class TestProjectFixture
```

Comment:
Maybe a summary comment that explains what this class's role in the world is would go a long way here.

---
In reply to: [66350938](https://github.com/dotnet/core-setup/pull/97#discussion_r66350938) [](ancestors = 66350938,66350076,66348239,66107984,66102620)